_data/sites.json

```
@@ -24341,6 +24341,16 @@
         ]
     },
 
+    {
+        "name": "Wilson Parking",
+        "url": "https://www.wilsonparking.com.au",
+        "difficulty": "hard",
+        "notes": "To delete your online prepaid account, download the Wilson Parking App, select 'Account', 'Your Profile' under the 'About You' header, then scroll to 'Delete Account'."
```

```suggestion
        "notes": "To delete your online prepaid account, download the Wilson Parking App, select 'Account', 'Your Profile' under the 'About You' header, then scroll to 'Delete Account'.",
```
it was missing a comma at the end of this sentence here.